src/support/timestamp.c

```diff
@@ -245,6 +245,57 @@ __wt_time_aggregate_validate(
 {
     char time_string[2][WT_TIME_STRING_SIZE];
 
+    /*
+     * The aggregated time window values that are tracked at the page level.
+     *    newest_start_durable_ts - The default value is WT_TS_NONE. It tracks the maximum durable
+     timestamp of all the inserts performed on a page.
+     *    newest_stop_durable_ts - The default value is WT_TS_NONE. It tracks the maximum durable
+     timestamp of all the removes performed on a page.
+     *    oldest_start_ts - The default value is WT_TS_NONE. It tracks the minimum commit timestamp
+     of any inserts performed on a page.
+     *    newest_txn - The default value is WT_TXN_NONE. It tracks the maximum transaction id of any
+     modification (insert/remove) performed on a page.
+     *    newest_stop_ts - The default value is WT_TS_MAX. It tracks the maximum commit timestamp of
+     a remove operation on a page. If there is no removal for a key, this value will be WT_TS_MAX.
+     *    newest_stop_txn - The default value is WT_TXN_MAX. It tracks the maximum commit
+     transaction id of a remove operation on a page. If there is no removal for a key, this value
+     will be WT_TXN_MAX.
+     *
+     *
+     * Three scenarios might happen at any point of time.
+     * Scenario 1 - No deletes on the page, only inserts and updates.
+     *    newest_stop_ts will be WT_TS_MAX since there is no removal of any key.
+     *    newest_start_durable_ts will be some valid value (not WT_TS_MAX or WT_TS_NONE)
+     *    newest_stop_durable_ts will be WT_TS_NONE
+     *    oldest_start_ts will be the minimum commit timestamp of any inserts performed on a page.
```

Comment:
could we have both start together and stop after wards.

Reply:
Fixed in d0719a967c76f22f12ccd16c910bb1b8acd8522b